MusicStore.DataAccess/Realization/UserAccountRepository.cs

```diff
@@ -58,5 +63,33 @@ public int Update(User item)
             }
             return -1;
         }
+
+        public bool CreateWithGuidId(string Id)
+        {
+            User user = new User()
+            {
+                IdentityKey = Id
+            };
+            _dataBase.Users.Add(user);
+            _dataBase.SaveChanges();
+            return true;
+        }
+
+        public int GetUserId(string identityId)
+        {
+            if (identityId == null)
+            {
+                throw new ArgumentException("Invalid user identity id");
+            }
+
+            var userId = _dataBase.Users.Where(x => x.IdentityKey == identityId).Select(x => x.Id);
+
+            if (userId.Count() == 1)
```

Comment:
Change to be Any()
It checks if there is **at least one** row in list.
Count is bad, because it will go through **every** element in collection and count it. Try to avoid using it.

Reply:
But i need to be confident that i have only one userId, any will not catch situation, when i will have more then one userId